layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationNodeDrawer.vue

```diff
@@ -281,7 +281,7 @@ export default {
       selectedPage: null,
       pageToEdit: false,
       link: '',
-      linkRules: [url => !!(url?.match(/^((https?:\/\/)?(www\.)?[a-zA-Z0-9:._\\/+=-]+\.[^\s]{2,})|(javascript:)|(\/portal\/)/))
+      linkRules: [url => !!(url?.match(/^((https?:\/\/)?(www\.)?[a-zA-Z0-9:._\\/+=-]+\.[^\s]{2,})|(javascript:)|(\/portal\/)|(\/meeds\/)/))
```

Comment:
do you mean Meta Site Name when adding `meeds` ? What if the meta site name changes ? (like in eXo bundle which becomes `dw`)

```suggestion
      linkRules: [url => !!(url?.match(/^((https?:\/\/)?(www\.)?[a-zA-Z0-9:._\\/+=-]+\.[^\s]{2,})|(javascript:)|(\/portal\/)|(\/meeds\/)/))
```